### PR TITLE
Docs panel: add support for loading images from the Cloud

### PR DESCRIPTION
**Pull request by MrFlashAccount**
Closes: https://github.com/enso-org/cloud-v2/issues/1618

This PR adds support for loading images from the Cloud.

### CODE REVIEW

File: app/common/src/text/english.json [hunk 1]
```diff
@@ -1,6 +1,10 @@
 {
   "submit": "Submit",
   "retry": "Retry",
+
+  "arbitraryFetchError": "An error occurred while fetching data",
+  "arbitraryFetchImageError": "An error occurred while fetching image",
```
Comment (somebody1234) [inline app/common/src/text/english.json]: `image` -> `an image`

File: app/gui/src/dashboard/components/MarkdownViewer/defaultRenderer.ts [hunk 2]
```diff
@@ -0,0 +1,46 @@
+/** @file The default renderer for Markdown. */
+import type { RendererObject } from 'marked'
+import { BUTTON_STYLES, TEXT_STYLE } from '../AriaComponents'
+
+/** The default renderer for Markdown. */
+export const defaultRenderer: RendererObject = {
```
Comment (somebody1234) [inline app/gui/src/dashboard/components/MarkdownViewer/defaultRenderer.ts]: would prefer `CONSTANT_CASE` for this i think

Comment (somebody1234) [reply app/gui/src/dashboard/components/MarkdownViewer/defaultRenderer.ts]: also optionally `Readonly<RendererObject>`

**Review by somebody1234 — APPROVED** [review]
CR :white_check_mark:

File: app/common/src/text/english.json (hunk 1 above)
Comment (MrFlashAccount) [reply app/common/src/text/english.json]: Maybe, it should be `the` image? 🤔

Comment (MrFlashAccount) [reply app/common/src/text/english.json]: But I don't think it's a big deal here

Comment (somebody1234) [reply app/common/src/text/english.json]: yeah this should probably be `the`. but yeah probably not a big deal

**Review by 4e6 — APPROVED** [review]

**Comment by PabloBuchu** [comment]
Nice job!

Screenshot 2024-12-20 at 15 16 14

**Review by PabloBuchu — APPROVED** [review]

**Pull request merged**